### ISSUE TITLE
Crashlog - Minecraft 1.16.3 with Alpha 6

### ISSUE THREAD

**Issue by seriousfreezing** [CompactMods/CompactMachines#447]
**Compact Machines version**:  compactmachines-4.0.0-alpha.6
**Forge version**: forge-1.16.3-34.1.41
**I am not using Optifine**: Yes, i am using Optifine.
**Link to Crashlog**: https://pastebin.com/y9Cdn7BZ
**Screenshot (if possible)**: no?

**Description of the problem including expected versus actual behavior:**
It was to open Minecraft and enter the game normally.

I open the game and it just closes out of nowhere, during loading.

**Steps to reproduce:**

1. open the Minecraft 1.16.3 with compactmachines-4.0.0-alpha.6, Optifine and Forge.
2. wait for loading.
3. crash :)

**Comment by robotgryphon** [comment]
Two things that I'm noticing at first glance.

1. Optifine does like to mess with mods, but I don't think it's at fault here. Just noting that if my next step does not work, try removing it.
2. Judging by the snippet below, you might have a duplicate mod jar in your folder. Can you post a screenshot of the `mods` directory?

```
[21:24:23] [main/INFO] [op.OptiFineTransformationService/]: OptiFineTransformationService.initialize
[21:24:25] [main/ERROR] [ne.mi.fm.lo.LanguageLoadingProvider/]: Missing language javafml version [35,) wanted by compactmachines-4.0.0-alpha.6.jar, found 34.1
Exception in thread "main" [21:24:25] [main/INFO] [STDERR/]: [java.lang.ThreadGroup:uncaughtException:1052]: java.lang.IllegalStateException: Duplicate key net.minecraftforge.fml.loading.moddiscovery.ModFileInfo@51df223b
```

**Comment by seriousfreezing** [comment]
Here: https://prnt.sc/xzuxu8 and https://prnt.sc/xzuyce

**Comment by robotgryphon** [comment]
Okay. I'm not seeing an obvious duplicate file in there, so I have two other guesses. 
1. Optifine is incompatible with your Forge version (just remove the mod for a moment to test that)
2. I accidentally have the mod loader forcing 1.16.4+. I thought Forge was allowing it, but that might be on me.
 
As an aside, I recommend updating to 1.16.5 anyway, since there's no difference for mods between the two versions. Most mods are already moving to 16.5 since they're compatible anyway.

**Comment by seriousfreezing** [comment]
I've been thinking about updating my version to 1.16.5 for a long time but I'm VERY afraid that I will lose some of the mods in this modpack, you know? Do you consider that I wait a while to update or do I do it right now?

**Comment by robotgryphon** [comment]
I feel that you should go for the update. There are some nasty server fixes that Mojang fixed in 16.5, and as far as Forge modders are concerned, it's basically an invisible update. If the modders are lagging behind, give 'em a nudge. All it should take is some metadata updating (on Curseforge) and at worst a recompile of their mods.

**Comment by seriousfreezing** [comment]
Thanks, but I didn't understand the metadata update part in Curseforge and recompiling my mods.
